### PR TITLE
feat: add go-to definition

### PR DESCRIPTION
**Pull request by vetlek**
feat: add go-to declaration

feat: add find references

### CODE REVIEW

File: server/src/parser/septicCnfg.ts [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+                location: {
+                    start: obj.identifier.start,
+                    end: obj.identifier.end,
+                },
+                obj: obj,
+            };
+            xvrRefs.push(ref);
+        }
+    }
+    switch (obj.type) {
```
Comment (einarsi) [inline server/src/parser/septicCnfg.ts:233]: Would this be better to put in the yaml file?

**Review by einarsi — COMMENTED** [review]
Just the one comment for now. Need to test functionality, and would be good with a run-through after.

File: server/src/parser/septicCnfg.ts (hunk 1 above)
Comment (vetlek) [reply server/src/parser/septicCnfg.ts]: Updated to get this information from the yaml file.

**Comment by vetlek** [comment]
Closes #100 
Closes #101

**Review by einarsi — APPROVED** [review]

**Pull request merged**